### PR TITLE
Issue 391

### PR DESCRIPTION
**Pull request by elma13**
For the choose tab, the data availability is checked in the measurements table, and directly shown in a calender plot. #391 
And some fix if there is no data in database, that the app still runs #396

### CODE REVIEW

**Comment by elma13** [comment]
@jspijker this is major adjustment in how to give information about availability to the user. In #397  I will continue adding the map too to the choose tab, and that the data is only gathered once from the cache. So that is the next step.

File: modules/show_data_avail_cache.R [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+      # Create visualisation of the available data in cache
+      # If no data at all
+      if(purrr::is_empty(stations_name)){
+        log_trace("mod show avail: Empty stations_name. No data available in cache ")
+
+        # message to the user
+        output$data_avail_or_not <- renderText({
+          "No data found, please consider to get them."
+        })
```
Comment (jspijker) [inline modules/show_data_avail_cache.R:56]: Translation?

**Review by jspijker — CHANGES_REQUESTED** [review]
Works, but I get English messages instead of Dutch. Should we fix this before next release or later?

**Comment by elma13** [comment]
@jspijker, I updated the translation for (I hope) all the modules. And added the info to the manual of function get_location_coordinates

**Review by jspijker — APPROVED** [review]
data availability works, Still some minor issues with i18n.

**Pull request merged**